lxd/device/device_utils_network.go

```diff
@@ -486,7 +486,7 @@ func networkNICRouteDelete(routeDev string, routes ...string) {
 }
 
 // networkSetupHostVethLimits applies any network rate limits to the veth device specified in the config.
-func networkSetupHostVethLimits(m deviceConfig.Device) error {
+func networkSetupHostVethLimits(d *deviceCommon, m deviceConfig.Device) error {
```

Comment:
As we're now passing `deviceCommon` in here, we don't actually need the `m` argument and can use `d.config` instead.

Reply:
This change should be a separate commit though to make backporting easier.

Reply:
Have done. Thanks!